### ISSUE TITLE
how to get reg_loss  in eager mode

### ISSUE THREAD

**Issue by Johnson-yue** [tensorflow/tensorflow#21237]
https://github.com/tensorflow/tensorflow/blob/9e0b05bbc4bb88d1b34fb2147429dc4ad7bd25cd/tensorflow/contrib/eager/python/examples/densenet/densenet_test.py#L105

In this demo, the Conv2D have regluarizer configure but when it computed loss, there is only softmax_loss!!
where reg_loss?????

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by lxuechen** [comment]
Hi Johnson,

Thanks a lot for reaching out! Yash wrote this example model, so I'm not totally sure of the details. But giving it a glance I think you're correct. To account for the regularization terms we would need to sum over all the terms in `model.losses`. I will confirm with Yash on this and get to you real soon.

Chen

**Comment by lxuechen** [comment]
Hi Johnson, 

We have updated the code. As a bit of explanation, `model.losses` returns a list of loss tensors (this is what `tf.keras.Model` do, not eager specific), so we need only add this to the cross entropy within the tape context.

Hope this helped!

Chen

**Comment by Johnson-yue** [comment]
@lxuechen Thank you very much. I will try it now.
